app/views/assignment_journals/_form.html.slim

```diff
@@ -0,0 +1,34 @@
+= simple_form_for [@assignment, @assignment_journal] do |f|
+  = simple_error_notice f
+
+  .row
+    .col-xs-12
+      = f.hidden_field :assignment_id, value: params[:assignment_id]
+
+  .row
+    .col-xs-12
+      = f.hidden_field :volunteer_id, value: @assignment.volunteer.id
```

Comment:
Since the hidden fields are invisible I would remove the `.row` and `.col` containers